src/apps/loan-list/modal/material-details-modal.tsx

```diff
@@ -1,3 +1,4 @@
+//  Todo: Fix infinite loop in MaterialDetailsModal
```

Comment:
Bahahaha. How fun. 
Try to describe it better what needs to be fixed so that people who don't know about the bug at all would be able to reproduce it 😉